src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -4,23 +4,35 @@
 
 * Update OTel API version to `1.4.0-beta.2`.
 
-  OTel API updated System.Diagnostics.DiagnosticSource to version 7.0.0
-  since [1.4.0-alpha.2](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.4.0-alpha.2).
+* Change ObservableGauge to ObservableUpDownCounter for the below metrics (which
+  better fit UpDownCounter semantics as they are additive.)
+
+  "process.runtime.dotnet.gc.heap.size",
+  "process.runtime.dotnet.gc.heap.fragmentation.size",
+  "process.runtime.dotnet.thread_pool.threads.count",
+  "process.runtime.dotnet.thread_pool.queue.length",
+  "process.runtime.dotnet.timer.count",
+  "process.runtime.dotnet.assemblies.count"
+  ([#675](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/675))
 
-  With this update, applications targeting .NET 5 and lower will receive a
-  warning at build time as described [here](https://github.com/dotnet/runtime/pull/72518)
-  (note: building using older versions of the .NET SDK produces an error at
-  build time). This is because .NET 5 reached EOL in May 2022 and .NET
-  Core 3.1 reaches EOL in December 2022.
+  If your backend system distinguishes between ObservableUpDownCounter and
+  ObservableGauge, then you may need to adjust your queries. Systems like
+  Prometheus are unaffected by this.
 
-  There is no guarantee that System.Diagnostics.DiagnosticSource will continue
-  to work on older versions of .NET. However, the build warning can be
-  suppressed by setting the SuppressTfmSupportBuildWarnings MSBuild property.
+* Removes NETCoreApp3.1 target as it is going out of support. The package keeps
```

Comment:
Might want to reword the first sentence slightly to be more clear. Something like...

> Now targets `net6.0` instead of `netcoreapp3.1` because .NET Core 3.1 & .NET 5 are going out of support.